store/entity-store.html

```diff
@@ -84,93 +100,99 @@
 		// updating the UI more consistently when dependent entities change as a result of Siren
 		// actions.
 		fetch: function(entityId, token, bypassCache) {
-			const lowerCaseToken = token.toLowerCase();
-			const lowerCaseEntityId = entityId.toLowerCase();
+			return this._getToken(token).then(({cacheKey, tokenValue}) => {
+				const lowerCaseToken = cacheKey.toLowerCase();
+				const lowerCaseEntityId = entityId.toLowerCase();
 
-			var entity = this._initContainer(this._store, entityId, token);
-			if (!entity || bypassCache) {
+				var entity = this._initContainer(this._store, entityId, cacheKey);
+				if (!entity || bypassCache) {
 
-				var headers = new Headers();
-				token && headers.set('Authorization', 'Bearer ' + token);
-
-				if (bypassCache) {
-					headers.set('pragma', 'no-cache');
-					headers.set('cache-control', 'no-cache');
-				}
+					var headers = new Headers();
+					cacheKey && headers.set('Authorization', 'Bearer ' + tokenValue);
```

Comment:
Why `cacheKey &&` and not `tokenValue &&`. Also this check at all implies we can proceed without a token/token might be undefined? If that's the case then `tokenValue.split('.')` will blow up in `_getToken`

Reply:
Yes, good catch! Thanks. I'll fix that.